src/Apps/Order2/Routes/Checkout/Components/PaymentStep/Order2PaymentForm.tsx

```diff
@@ -404,21 +402,19 @@ const PaymentFormContent: React.FC<PaymentFormContentProps> = ({
       setIsSubmittingToStripe(true)
 
       try {
-        const orderOrError = (
-          await setPaymentMutation.submitMutation({
-            variables: {
-              input: {
-                id: order.internalID,
-                paymentMethod: "CREDIT_CARD",
-                paymentMethodId: selectedCreditCard?.internalID,
-              },
+        const result = await setPaymentMutation.submitMutation({
+          variables: {
+            input: {
+              id: order.internalID,
+              paymentMethod: "CREDIT_CARD",
+              // Note: paymentMethodId is not supported in updateOrder mutation
```

Comment:
I believe we need this for the saved CC flow to work, can you please test it locally to confirm?